.github/workflows/aes.yml

```
@@ -16,6 +16,7 @@ defaults:
 env:
   CARGO_INCREMENTAL: 0
   RUSTFLAGS: "-Dwarnings"
+  SDE_FULL_VERSION: "9.53.0-2025-03-16"
```

I noticed there's a slightly newer one, FWIW:
```suggestion
  SDE_FULL_VERSION: "9.58.0-2025-06-16"
```